src/lightning/pytorch/loops/evaluation_loop.py

```diff
@@ -380,6 +380,7 @@ def _evaluation_step(self, batch: Any, batch_idx: int, dataloader_idx: int) -> N
         """
         trainer = self.trainer
 
+        batch = trainer.precision_plugin.convert_input(batch)
         batch = trainer.lightning_module._on_before_batch_transfer(batch, dataloader_idx=dataloader_idx)
         batch = call._call_strategy_hook(trainer, "batch_to_device", batch, dataloader_idx=dataloader_idx)
```

Comment:
Is there a case for `batch_to_device` as a strategy hook after this PR?

Reply:
The base strategy calls it and calls other hooks, no other strategy overrides it